Specs/NumberWithUnit/English/CurrencyModel.json

```diff
@@ -1,1441 +1,1525 @@
-[{
+[
+ {
+  "Input": "it only cost 3 dollars 50 cents.",
+  "NotSupportedByDesign": "python",
+  "NotSupported": "javascript",
+  "Results": [
+   {
+    "Text": "3 dollars 50 cents",
+    "TypeName": "currency",
+    "Resolution": {
+     "value": "3.5",
+     "unit": "Dollar"
+    }
+   }
+  ]
+ }
+,
+ {
+  "Input": "it only cost thirteen dollars and forty-five cents",
+  "NotSupportedByDesign": "python",
+  "NotSupported": "javascript",
+  "Results": [
+   {
+    "Text": "thirteen dollars and forty-five cents",
+    "TypeName": "currency",
+    "Resolution": {
+     "value": "13.45",
+     "unit": "Dollar"
+    }
+   }
+  ]
+ }
+,
+ {
+  "Input": "it only cost thirteen dollars forty-five cents",
+  "NotSupportedByDesign": "python",
+  "NotSupported": "javascript",
+  "Results": [
+   {
+    "Text": "thirteen dollars forty-five cents",
+    "TypeName": "currency",
+    "Resolution": {
+     "value": "13.45",
+     "unit": "Dollar"
+    }
+   }
+  ]
+ }
+,
+ {
+  "Input": "it only cost thirteen dollars forty five",
+  "NotSupportedByDesign": "python",
+  "NotSupported": "javascript",
+  "Results": [
+   {
+    "Text": "thirteen dollars forty five",
+    "TypeName": "currency",
+    "Resolution": {
+     "value": "13.45",
+     "unit": "Dollar"
+    }
+   }
+  ]
+ }
+,
+ {
   "Input": "montgomery county , md . - - $ 75 million of general obligation , series b , consolidated public improvement bonds of 1989 , through a manufacturers hanover trust co . group .",
   "NotSupportedByDesign": "python",
   "Results": [
-    {
-      "Text": "$ 75 million",
```

Comment:
Nit: you should not change the indentation. All other specs follow the original one.